qiskit_machine_learning/neural_networks/estimator_qnn.py

```diff
@@ -0,0 +1,249 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Estimator quantum neural network class"""
+
+from __future__ import annotations
+
+from copy import copy
+import logging
+from typing import Sequence, Tuple
```

Comment:
the `tuple` annotation is included in the `__future__` `annotations`
```suggestion
from typing import Sequence
```

Reply:
And replace `Tuple` to `tuple` down below :)